packages/layout/src/css/page-heading-layout.js

```diff
@@ -2,7 +2,7 @@ import { layout } from '@pluralsight/ps-design-system-core'
 
 export default {
   '.psds-page-heading-layout': {
-    padding: `${layout.spacingLarge} 0`
+    padding: layout.spacingLarge
```

Comment:
is the horz padding not needed anymore?

Reply:
No I realized after publishing earlier merges a change I marked as breaking didn't bump to a new major. I moved this back so that people using that version would not experience breakage. I just created a new branch that will hopefully bump the version of layout to a new major.